WSRequestHandler.cpp

```diff
@@ -710,6 +712,58 @@ void WSRequestHandler::HandleGetMute(WSRequestHandler* req) {
     obs_data_release(response);
 }
 
+void WSRequestHandler::HandleSetSyncOffset(WSRequestHandler* req) {
+    if (!req->hasField("source") ||
+	!req->hasField("offset")) {
+	req->SendErrorResponse("missing request parameters");
+	return;
+    }
+
+    const char* source_name = obs_data_get_string(req->data, "source");
+    int64_t source_sync_offset = (int64_t)obs_data_get_int(req->data, "offset");
+
+    if (source_name == NULL || strlen(source_name) < 1 ||
```

Comment:
It's C++11 code, so use `nullptr` instead of `NULL`. Or better: use `!source_name` as the condition instead, like below with `!source`.